src/main/java/frc/robot/commands/KsCharacterization.java

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.SwerveSubsystem;
+import frc.robot.swerve.SwerveModuleAbstract;
+
+public class KsCharacterization extends Command {
```

Comment:
What's "KS"? Would it worth spelling out for clarity?

Reply:
done